File: 30-Days-of-Code/Day-2-Operators/PHP/solution.php
<?php
$handle = fopen ("php://stdin","r");

$mealCost;
$tipPercent;
$taxPercent;

$mealCost = fgets($handle);
$tipPercent = fgets($handle);
$taxPercent = fgets($handle);

$tip = $mealCost * ($tipPercent / 100);
$tax = $mealCost * ($taxPercent / 100);
$totalCost = $mealCost + $tip + $tax;

$roundedTotalCost = round($totalCost);


printf ("The total meal cost is %d dollars.\n", $roundedTotalCost);

fclose($handle);
?>

File: 30-Days-of-Code/Day-2-Operators/OCaml/solution.ml
let mealCost = float_of_string(input_line stdin);;
let tipPercent = int_of_string (input_line stdin);;
let taxPercent = int_of_string (input_line stdin);;

let tip = mealCost *. (float_of_int (tipPercent) /. 100.);;
let tax = mealCost *. (float_of_int (taxPercent) /. 100.);;
let totalCost = mealCost +. tip +. tax;;

let roundedTotalCost = int_of_float(floor (totalCost +. 0.5));;


Printf.printf "The total meal cost is %d dollars.\n" roundedTotalCost;;
